Makefile

```diff
@@ -42,7 +42,7 @@ uninstall: ## Remove executable from PATH
 	rm -f $(DESTDIR)$(prefix)$(mandir)/man1/editorconfig-checker.1
 
 test: ## Run test suite
-	@go test -race -coverprofile=coverage.txt -covermode=atomic ./...
+	@go test -ldflags "-X github.com/editorconfig-checker/editorconfig-checker/v3/cmd/editorconfig-checker.version=$(CURRENT_VERSION)" -race -coverprofile=coverage.txt -covermode=atomic ./...
```

Comment:
Could you explain why this change is needed?

Reply:
This is one of the changes I had to make due to the issue I described in #383 - since with that PR we are starting to run the binary (or at least it's `main` function) we also tend to parse config files - which means we need to have the current version available for comparison - hence the injection of the overwriting the initializing value of that variable.

Reply:
Alright, thank you for the explanation. :+1: